cmd/eib/main.go

```diff
@@ -22,20 +24,34 @@ func init() {
 	log.SetOutput(os.Stdout)
 }
 
-func processArgs() (*config.ImageConfig, error) {
+func processArgs() (*config.ImageConfig, *config.BuildConfig, error) {
```

Comment:
Nit: The more things we add to this function, the more I'd like to just remove it and inline it in main().